src/os-freebsd.h

```diff
@@ -19,6 +19,10 @@
 
 #define INADDR_ALLIGMPV3_GROUP ((in_addr_t) 0xe0000016)
 
+#ifndef LOGIN_NAME_MAX
+#define LOGIN_NAME_MAX 256
+#endif
```

Comment:
This looks like fully unrelated change to igmpv3, so it should not be in pull request for sending igmpv3 queries. Please move it into separate bug issue / pull request.